src/GitHub.jl

```diff
@@ -325,4 +325,16 @@ export
     create_release,
     releases
 
+############
+# Licenses #
+############
+
+include("licences/license.jl")
```

Comment:
You gotta choose one spelling and stick to it 😅 Looks like `license` is what GitHub uses.

Reply:
I tried so hard to stick to license :D Good catch :)